src/SourceBuild/content/eng/finish-source-only.proj

```diff
@@ -226,8 +225,8 @@
           DependsOnTargets="DetermineSourceBuiltSdkNonStableVersion;ResolveProjectReferences">
     <!-- Inputs: Packages to include in the tarball -->
     <ItemGroup>
-      <ArtifactsPackageToBundle Include="$(ArtifactsShippingPackagesDir)**;
-                                          $(ArtifactsNonShippingPackagesDir)**" />
+      <ArtifactsPackageToBundle Include="$(ArtifactsTmpDir)/staging/packages/$(Configuration)/Shipping/**;
+                                          $(ArtifactsTmpDir)/staging/packages/$(Configuration)/NonShipping/**" />
```

Comment:
Why does this need to change? Isn't `ArtifactsShippingPackagesDir` and `ArtifactsNonShippingPackagesDir` populated anymore?

Reply:
These folders aren't the `ArtifactsShippingPackagesDir` and `ArtifactsNonShippingPackagesDir` paths as they're grabbing the post-published files.

The previous implementation would capture all packages, even if they're Vertical visibility only, and put them into the tarball.